### PR TITLE
Improve NL translation

### PR DESCRIPTION
**Pull request by yliefting**
Redacted Dutch translation. The privacy and security sections still need translations but these were skipped to prevent possible legal issues with the wording.

### CODE REVIEW

File: app/locales/nl.js [hunk 1]
```diff
@@ -34,82 +34,82 @@ export default {
...
-      taskTab: 'Task',
-      tutorialTab: 'Tutorial'
+      taskTab: 'Taak',
+      tutorialTab: 'Uitleg'
     },
-    recents: 'Je recente classificaties',
-    talk: 'Praat',
-    taskHelpButton: 'Need some help with this task?',
-    miniCourseButton: 'Herstart de minitour'
+    recents: 'Uw recente waarnemingen',
```
Comment (marten) [inline app/locales/nl.js]: Up for discussion: do we want to use the Dutch formal "you" (`u`) or the less-formal "you" (`je`)?

Comment (marten) [reply app/locales/nl.js]: @mrniaboc might be the best to decide this i guess?

**Comment by eatyourgreens** [comment]
This PR is falling behind changes in master and should be merged or closed. @mrniaboc @marten any strong opinions on whether project language should address volunteers formally or informally?

**Comment by marten** [comment]
My personal preference would be informal --- I don't see us as a very formal bunch. 

- Marten

**Comment by yliefting** [comment]
I can rewrite the text in the informal form, if that's the preferred way of addressing users.

**Comment by srallen** [comment]
Since @marten was the only to express a preference, @yliefting could you rewrite in the informal? Thanks!

**Comment by yliefting** [comment]
Sure, I'll try to finish that this week.

**Pull request merged**